### DISCUSSION TITLE
Can this tool alter stuff for steam deck's bios?

### DISCUSSION THREAD

**Discussion by DuckersMcQuack** [DavidS95/Smokeless_UMAF#67] [Q&A]
As i want to add options in uma frame buffer settings to be able to choose higher than 4GB uma frame buffer. Preferably 6 and 8GB, as i have a python experiment i want to do on the steam deck, but it's limited by the uma buffer.

**Comment by TDuffinNTU** [comment]
Yep! This tool is featured by Cryobyte33 in a video regarding undervolting your deck (which was since added as an official BIOS option)

You'll need a USB hub to insert a USB stick *and* a keyboard. Smokeless doesn't support the Steamdeck's controller buttons

https://www.youtube.com/watch?v=LNEI7BTc87Q

**Reply by DARTH-BARSI** [reply]
it seems to be broken with the latest steam deck bios as device manager doesnt show properly

**Reply by DuckersMcQuack** [reply]
I'm not going to OC or undervolt, i will simply just alter the uma buffer with additional options for 6 and 8GB shared Vram

**Reply by DARTH-BARSI** [reply]
Your best with cryoutilities rather than this for what you want
https://github.com/CryoByte33/steam-deck-utilities
